_includes/troubleshoot/issues/cr-create-no-action.md

```diff
@@ -0,0 +1,42 @@
+## StorageOS resources not created
+
+### Issue:
+
+StorageOS resources such as NS, DaemonSets or Pods are not being created after
+creating the ClusterOperator CustomResource.
+
+```bash
+{{ page.cmd }} -n storageos get all
+No resources found.
+```
+
+### Reason:
+Most likely you forgot to create the StorageOS secret before creating the
+CustomResource.
+
+
+### Assert
+Check the events in the NameSpace you created the CR.
```

Comment:
Can we s/CR/CustomResource/ everywhere? It's a little disconcerting to those not intimately familiar with Rancher.

Reply:
CR are K8S, not rancher specific. I changed all CR for CustomResource as you mentioned, it is a term that not many people will be familiar with.